app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -143,13 +146,16 @@ module?.exports = React.createClass
       <div>
         {selectKeys.map (i) =>
           options = @getOptions(i)
+          disabled = @getDisabledAttribute(i)
           <div key={Math.random()}>
-            <div>{selects[i].title}</div>
+            {if selects[i].title isnt @props.task.instruction
+              <div>{selects[i].title}</div>}
```

Comment:
Is this `if` checking for something specific? Seems kinda weird to just leave out the select title.